src/pipeline.ml

```diff
@@ -222,6 +224,11 @@ let v ~app ~notify:channel ~sched ~staging_auth () =
         docker "docker/storage/Dockerfile"  ["live", "ocurrent/docs-ci-storage-server:live", [`Ci6, "infra_storage-server"]];
         docker "Dockerfile.web"             ["live-web", "ocurrent/docs-ci-web:live", [`Ci6, "infra_docs-ci-web"]];
       ];
+      ocurrent, "current-bench", [
+        docker "pipeline/Dockerfile" ["live", "ocurrent/current-bench-pipeline", [`Cb, "current-bench-pipeline"]];
```

Comment:
Are you sure these service names are correct? There is normally a `_` somewhere, separating the stack name from the service name.